website/blog/2024-06-26-llm-as-judge/index.md

```diff
@@ -0,0 +1,522 @@
+            score=2,
+            justification="This is a literal translation that doesn't capture the idiomatic meaning."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="Break a leg!",
+            output="¡Mucha mierda!",
+            score=5,
+            justification="This translation uses the equivalent Spanish theater idiom, showing high cultural awareness."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="It's raining cats and dogs.",
+            output="Está lloviendo gatos y perros.",
+            score=1,
+            justification="This literal translation does not convey the idiomatic meaning of heavy rain."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="It's raining cats and dogs.",
+            output="Está lloviendo a cántaros.",
+            score=5,
+            justification="This translation uses a Spanish idiom that accurately conveys the meaning of heavy rain."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="Kick the bucket.",
+            output="Patear el balde.",
+            score=1,
+            justification="This literal translation fails to convey the idiomatic meaning of dying."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="Kick the bucket.",
+            output="Estirar la pata.",
+            score=5,
+            justification="This translation uses the equivalent Spanish idiom for dying, showing high cultural awareness."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="Once in a blue moon.",
+            output="Una vez en una luna azul.",
+            score=2,
+            justification="This literal translation does not capture the rarity implied by the idiom."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="Once in a blue moon.",
+            output="De vez en cuando.",
+            score=4,
+            justification="This translation captures the infrequency but lacks the idiomatic color of the original."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="The ball is in your court.",
+            output="La pelota está en tu cancha.",
+            score=3,
+            justification="This translation is understandable but somewhat lacks the idiomatic nuance of making a decision."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="The ball is in your court.",
+            output="Te toca a ti.",
+            score=5,
+            justification="This translation accurately conveys the idiomatic meaning of it being someone else's turn to act."
+        )
+    ],
+    model="openai:/gpt-4",
+    parameters={"temperature": 0.0},
+)
+
+```
+
+On top of this let's use mlflow default metrics for the evaluators. In this case mlflow wll use roberta-hate-speech model to detect the toxicity
```

Comment:
punctuation + briefly explain the concept of toxicity and why it might be relevant to this example